### PR TITLE
Add NUC11 docs

### LINKED ISSUE #307: Add NUC11 docs
SD 2.2.0 will add NUC11 support (https://github.com/freedomofpress/securedrop/issues/6170). We should document this similar to  https://docs.securedrop.org/en/stable/hardware.html#intel-10th-gen-nuc , which means we also have to:
- confirm that wireless removal docs are still accurate (requires opening up the device)
- confirm that BIOS instructions are still accurate

### PR DESCRIPTION
**Pull request by eaon**
## Status

Ready for review

## Description of Changes

* Description: With 2.2.0 we'll support NUC11s. Checked with @zenmonkeykstop beforehand to make sure the model we recommend does not support SGX, but added a note for other models regardless.

* Fixes #307

## Release 

* 2.2.0

## Checklist (Optional)

- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

**Review by eloquence — APPROVED** [review]
LGTM based on visual review. @cfm , @zenmonkeykstop or other NUC11 owners, could you do final diligence check before merge?

File: docs/hardware.rst [hunk 1]
```diff
@@ -392,7 +392,7 @@ Specific Hardware Recommendations
 Application and Monitor Servers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We currently recommend Intel 7-, 8-series and 10-series NUCS for SecureDrop
+We currently recommend Intel 7-, 8-, 10-series and 11-series NUCs for SecureDrop
```
Comment (cfm) [inline docs/hardware.rst:395]: Nit for easier scanning:  :-)

```suggestion
We currently recommend Intel 7-, 8-, 10-, and 11-series NUCs for SecureDrop
```

File: docs/hardware.rst [hunk 2]
```diff
@@ -414,6 +414,38 @@ NUC before it can be used. We recommend:
...
+|NUC11 leads|
+
+Before the initial OS installation, boot into the BIOS by pressing **F2** at
+startup and adjust the system configuration:
+
+- Under **Advanced ▸ Onboard Devices**, disable all onboard devices
+  other than LAN: HD audio, microphone, Thunderbolt, WLAN, Bluetooth,
+  SD card controller, and enhanced consumer infrared.
+
+- Under **Boot ▸ Secure Boot**, uncheck the **Secure Boot** checkbox.
```
Comment (cfm) [inline docs/hardware.rst:439]: @eaon, does your NUC's version of the Intel Visual BIOS expose the **Secure Boot** setting as a checkbox?  I have a dropdown menu, in which the desired setting is `Disabled`.

**Review by cfm — APPROVED** [review]
BIOS instructions are easy to follow on my NUC11s.  I've noted two nits, one prose- and one instruction-related, but I'm approving on the substance.  :-)

File: docs/hardware.rst (hunk 2 above)
Comment (eaon) [reply docs/hardware.rst:439]: You are correct! Thanks for catching!

**Review by cfm — APPROVED** [review]
Looks great, @eaon!  I'll defer to @eloquence as documentation maintainer for the merge.

**Pull request merged**